index.html

```diff
@@ -0,0 +1,242 @@
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Come and participate at our products exhibition, and
+                    </p>
+                </div>
+
+            </div>
+
+
+            <div class="event-card">
+                <div class="trailing">
+                    <div class="">
+                        <img src="assets/images/event-vect3.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h1>Forum</h1>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Have the time to share your thoughts and opinions with experts for each topic.
+                    </p>
+                </div>
+
+            </div>
+
+            <div class="event-card">
+                <div class="trailing">
+                    <div class="">
+                        <img src="assets/images/event-vect4.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h1>Workshop</h1>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Try creating your own work using open source license rather than just watching at it.
+                    </p>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="event-btn-container">
+            <button>Join CC Global Summit 2015</button>
+        </div>
+
+        <div class="lg-event-link-container">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </div>
+
+    </section>
+
+
+    <section class="mobile-team">
+        <div class="team-mobile-list-wrapper">
+            <h1 class="expert-title">Our experts</h1>
+            <ul class="mobile-team-list">
+
+            </ul>
+        </div>
+
+        <button class="team-btn">
+            <p>MORE</p>
+            <i class='bx bxs-chevron-down'></i>
+        </button>
+    </section>
+
+    <section class="lg-screen-team">
+        <h1 class="expert-title">Our experts</h1>
+        <ul class="mobile-team-list">
+
+        </ul>
+    </section>
+
+    <footer class="home-footer">
+        <section class="partner">
+            <h1>Partner</h1>
+            <div class="partner-container">
+                <img src="assets/images/Rolex.svg" alt="">
+                <img src="assets/images/Gucci-Logo.wine.svg" alt="">
+                <img src="assets/images/Chanel_logo.svg" alt="">
+                <img src="assets/images/Zara_Logo.svg" alt="">
+            </div>
+        </section>
+
+        <div class="mobile-footer-container">
+            <img src="assets/images/logo.png" alt="">
+
+            <div class="">
+                <p>Yoka-Look<br /> All Rights Reserved.</p>
+            </div>
+        </div>
+    </footer>
+</body>
```

Comment:
Currently, the footer is not visible on the mobile screen, kindly ensure it is visible so that your design matches the template.